components/eamxx/scripts/test_all_scream.py

```diff
@@ -1,8 +1,6 @@
 from utils import run_cmd, run_cmd_no_fail, expect, check_minimum_python_version, ensure_psutil, \
     SharedArea, safe_copy
-from git_utils import get_current_head, get_current_commit, get_current_branch, is_repo_clean, \
```

Comment:
I wonder if we should rm the git utils we no longer use, and keep the code base leaner...